packages/core/src/components/BulkActions/BulkActions.tsx

```diff
@@ -54,7 +54,7 @@ export interface HvBulkActionsProps extends HvBaseProps {
   /**
    * Function called when the "select all pages" button is clicked toggled.
    */
-  onSelectAllPages?: HvButtonProps["onClick"];
+  onSelectAllPages?: HvButtonProps<"button">["onClick"];
```

Comment:
We shouldn't need to specify `<"button">` every time. We can default it to `"button"`

```suggestion
  onSelectAllPages?: HvButtonProps["onClick"];
```